migrations/sqlite/20240530125008_event_header.up.sql

```diff
@@ -0,0 +1,9 @@
+-- Add up migration script here
+CREATE TABLE IF NOT EXISTS "ceramic_one_event_header" (
+    "cid" BLOB NOT NULL, -- event cid
+    "event_type" INTEGER NOT NULL, -- enum EventType: Init, Data, Time
+    "stream_cid" BLOB NOT NULL,  -- id field in header. can't have FK because stream may not exist until we discover it but should reference ceramic_one_stream(cid)
+    "prev" BLOB, -- prev event cid (can't have FK because node may not know about prev event)
+    PRIMARY KEY(cid),
+    FOREIGN KEY(cid) REFERENCES ceramic_one_event(cid)
```

Comment:
I didn't want to make that table any wider as we do large scans and we have to load the entire row off disk even if we only read some of the columns. Since these types are pretty small it probably doesn't matter, but I kind of expected `event_metadata` (probably going with that since `event_header` is silly) to keep growing and I didn't want to end up with a 25 column event table.

If I could redefine these tables (wow, that was quick), I'd rename event -> event_hash and make event_header/event_metadata the event table and put delivered and all the new data in there.

Reply:
okay fair enough.  Was also just talking to @smrz2001 about the fact that for self-anchoring we'll probably want to track whether we learned of an event via the API or via Recon.  That might be something we want to add to this metadata table as well.